api/v1beta1/ssp_types.go

```diff
@@ -53,6 +53,27 @@ type NodeLabeller struct {
 	Placement *lifecycleapi.NodePlacement `json:"placement,omitempty"`
 }
 
+type CommonInstancetypes struct {
+	// URL of a Kustomize remote target from which to generate and deploy resources.
+	//
+	// The following caveats apply to the provided URL:
+	//
+	// * 'file://' URLs are not supported and will be rejected by the SSP validation webhook.
+	//
+	// * The URL must include '?ref=$ref' pinning it to a specific reference. It is recommended
```

Comment:
Can you add that referencing a branch will have unpredictable reconcile behavior?

Reply:
I've basically said as much but I'll make it more explicit yeah.